llvm/lib/Target/AArch64/AArch64InstrGISel.td

```diff
@@ -274,6 +274,11 @@ def : GINodeEquiv<G_EXTRACT_VECTOR_ELT, vector_extract>;
 
 def : GINodeEquiv<G_PREFETCH, AArch64Prefetch>;
 
+def : GINodeEquiv<G_VECREDUCE_UMIN, vecreduce_umin>;
```

Comment:
This should be in llvm/include/llvm/Target/GlobalISel/SelectionDAGCompat.td, as it is a generic node.